src/main/java/edu/greenblitz/tobyDetermined/commands/TelescopicArm/Commands/Elbow/RotateToAngle.java

```diff
@@ -0,0 +1,42 @@
+package edu.greenblitz.tobyDetermined.commands.TelescopicArm.Commands.Elbow;
+
+import edu.greenblitz.tobyDetermined.commands.TelescopicArm.ElbowCommand;
+import edu.wpi.first.math.util.Units;
+
+public class RotateToAngle extends ElbowCommand {
+
+    private double wantedAngle;
+    private int isOnAnglecnt;
+    private static final int inAngleTime = 5; //in ticks
```

Comment:
first of all ticks isnt the correct name for the units here
secondly this system should not be fast enough to require timed tolerance this can be removed(in the subsystem)

Reply:
what should i do?